bridge-fungible-token/tests/utils/environment.rs

```diff
@@ -1,33 +1,55 @@
 use crate::utils::builder;
 
-use std::mem::size_of;
-use std::num::ParseIntError;
-use std::result::Result as StdResult;
-use std::str::FromStr;
-
-use fuels::signers::fuel_crypto::SecretKey;
-use fuels::test_helpers::{setup_single_message, setup_test_client, Config, DEFAULT_COIN_AMOUNT};
-use fuels::tx::{
-    Address, AssetId, Bytes32, ConsensusParameters, Input, Output, Receipt, TxPointer, UtxoId, Word,
+use std::{mem::size_of, num::ParseIntError, result::Result as StdResult, str::FromStr};
+
+use fuel_core_types::{
+    fuel_tx::{Bytes32, Input, Output, Receipt, TxPointer, UtxoId},
+    fuel_types::Word,
 };
 use fuels::{
-    prelude::*,
+    accounts::{
+        fuel_crypto::SecretKey, predicate::Predicate, wallet::WalletUnlocked, Signer,
+        ViewOnlyAccount,
+    },
+    prelude::{
+        abigen, setup_custom_assets_coins, setup_test_provider, Address, AssetConfig, AssetId,
+        Bech32ContractId, Config, Contract, ContractId, LoadConfiguration, Provider,
+        ScriptTransaction, TxParameters,
+    },
+    test_helpers::{setup_single_message, DEFAULT_COIN_AMOUNT},
     types::{message::Message, Bits256},
 };
 use primitive_types::U256 as Unsigned256;
 use sha3::{Digest, Keccak256};
 
-fn keccak_hash<B>(data: B) -> Bytes32
-where
-    B: AsRef<[u8]>,
-{
-    let mut hasher = Keccak256::new();
-    hasher.update(data);
-    <[u8; Bytes32::LEN]>::from(hasher.finalize()).into()
-}
-
 const CONTRACT_MESSAGE_PREDICATE_BINARY: &str =
     "../bridge-message-predicates/contract_message_predicate.bin";
+const MESSAGE_SENDER_ADDRESS: &str =
+    "0x00000000000000000000000096c53cd98B7297564716a8f2E1de2C83928Af2fe";
+const TEST_BRIDGE_FUNGIBLE_TOKEN_CONTRACT_BINARY: &str =
+    "../bridge-fungible-token/out/debug/bridge_fungible_token.bin";
+const DEPOSIT_RECIPIENT_CONTRACT_BINARY: &str =
+    "../test-deposit-recipient-contract/out/debug/test_deposit_recipient_contract.bin";
+
+abigen!(
+    Contract(
+        name = "BridgeFungibleTokenContract",
+        abi = "./bridge-fungible-token/out/debug/bridge_fungible_token-abi.json",
+    ),
+    Predicate(
```

Comment:
This can be a separate issue but I just wanted to mention that we can remove predicate and script code now by including a dependency for the [predicate repo crate](https://crates.io/crates/fuel-contract-message-predicate). Then whenever we need the the bytecode for either the script or predicate you just use something like `fuel_contract_message_predicate::script_bytecode()`. The crate also provides functions for getting the predicate root quickly too (`fuel_contract_message_predicate::predicate_root`).

Reply:
I don't know what will happen to that crate as we move towards a monorepo. Either way, I've raised an issue to refactor the repo prior to that merge so I'll try that in the subsequent issue

Reply:
https://github.com/FuelLabs/fuels-rs/blob/6efa6f11c8f69f4ef604b70bb78d55f6d4bf76e1/Cargo.toml#L56